tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -72,33 +72,33 @@ Verify That Prometheus Image Is A CPaaS Built Image
     ...     Tier1
     ...     ODS-734    
     ${pod} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=prometheus-
```

Comment:
As pod is a list maybe it's better to call it ${pods}    (in all tests)
`${pods} =    Search Pod    namespace=redhat-ods-monitoring    pod_start_with=prometheus-`

Alternatively, you could create a new keyword  `Find First Pod By Name`